src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -1957,13 +1957,14 @@ s32 func_80832CB0(PlayState* play, Player* this, LinkAnimationHeader* anim) {
     }
 }
 
-void Player_SkelAnimeResetPrevTranslRot(Player* this) {
+void Player_ResetAnimMovement(Player* this) {
     this->skelAnime.prevTransl = this->skelAnime.baseTransl;
     this->skelAnime.prevRot = this->actor.shape.rot.y;
 }
 
-void Player_SkelAnimeResetPrevTranslRotAgeScale(Player* this) {
-    Player_SkelAnimeResetPrevTranslRot(this);
+void Player_ResetAnimMovementScaledByAge(Player* this) {
```

Comment:
Should "scaled by age" be considered the common case, so that "not scaled by age" gets a more verbose name instead? I notice `PLAYER_ANIM_MOVEMENT_RESET` is completely unused for example. Why would you want to use it anyway?

Reply:
The "scaled by age" version is literally "do the reset, then scale stuff by age". Im not sure it really makes sense to have it as the "base" version when it just isnt.

Its never implicitly called with the flag, but it will be used when `PLAYER_ANIM_MOVEMENT_RESET_BY_AGE` isnt specified and youre transitioning to another animation with movement. 

It probably makes sense to use if an animation is already scaled for child link (as in it was made for him) vs child link.using an existing adult link animation.

Reply:
if length of the function is a concern, I would be fine with `Player_ResetAnimMovementByAge`. This is what I named the flag already even, I think it can be inferred that "by age" is doing some sort of scaling

Reply:
I thought "scaled by age" was the default case since I didn't fully understand that `Player_ResetAnimMovement` is also called when no `PLAYER_ANIM_MOVEMENT_` are used